hassio/src/dashboard/hassio-addons.ts

```diff
@@ -70,35 +86,18 @@ class HassioAddons extends LitElement {
     ];
   }
 
-  private _computeIcon(addon: HassioAddonInfo): string {
-    return addon.installed !== addon.version
-      ? "hassio:arrow-up-bold-circle"
-      : "hassio:puzzle";
-  }
-
-  private _computeIconTitle(addon: HassioAddonInfo): string {
-    if (addon.installed !== addon.version) {
-      return "New version available";
-    }
-    return addon.state === "started"
-      ? "Add-on is running"
-      : "Add-on is stopped";
-  }
-
-  private _computeIconClass(addon: HassioAddonInfo): string {
-    if (addon.installed !== addon.version) {
-      return "update";
-    }
-    return addon.state === "started" ? "running" : "";
-  }
-
   private _addonTapped(ev: any): void {
     navigate(this, `/hassio/addon/${ev.currentTarget.addon.slug}`);
   }
 
   private _openStore(): void {
     navigate(this, "/hassio/store");
   }
+
+  private get _computeHA105plus(): boolean {
```

Comment:
Why do we calculate this even?
It seems to be only used to see if we are on Home Assistant 105, however, this is not tied into the Home Assistant Core, but the Supervisor. Furthermore, the icons are already available in the API for more than a year, since we push the Supervisor, it is pretty fair to say, there is no backward compatibility to take into account here.

Reply:
We need this because it is refenced as a img, those do not support authentication, 105 of Core made the icons available without authentication.

Reply:
Thanks for clearing that up! 👍